backend/src/main/java/cookme/controller/RecipesController.java

```diff
@@ -39,4 +42,32 @@ public void deleteRecipeWithId(@PathVariable String id) {
     recipesService.findRecipeById(id);
     recipesService.deleteRecipe(id);
     }
+    // User Favorites Endpoints
+
+    @PostMapping("/user/{userId}/favorites/{recipeId}")
```

Comment:
From a security perspective, it is not ideal to include the {userId} in the endpoint. Even though I have done it this way in my project as well. ;)

Reply:
i will change it as well